composer.json

```diff
@@ -10,10 +10,10 @@
         }
     ],
     "require": {
-        "php": ">=7.2",
-        "illuminate/console": "^5.5|^6.0|^7.0|^8.0",
-        "illuminate/contracts": "^5.5|^6.0|^7.0|^8.0",
-        "illuminate/support": "^5.5|^6.0|^7.0|^8.0",
+        "php": "^8.0",
```

Comment:
To keep compatibility with older versions it must remain ">=7.2"

Reply:
Is added now